scaling_performance/optimizing_storage.adoc

```diff
@@ -281,7 +281,9 @@ STORAGE_DRIVER=overlay2
 +
 [NOTE]
 ====
-If using CRI-O specify `STORAGE_DRIVER=overlay` to use overlay2.
+If using CRI-O specify `STORAGE_DRIVER=overlay`.
+
+Note: CRI-O the default "overlay" storage driver uses the overlay2 optimizations.
```

Comment:
s/Note: CRI-O the default "overlay"/With CRI-O, the default \`overlay\`

Reply:
Fixed